src/iotjs.cpp

```diff
@@ -166,8 +163,7 @@ static bool StartIoTjs(Environment* env) {
   // Release bulitin modules.
   CleanupModules();
 
-  // Release jerry null and undefined objects.
-  JObject::cleanup();
+  // Release commonly used jerry values
```

Comment:
I noticed that the original comments terminated by dot (.), but the new comments does not. Is this an intentional change?